packages/utils/src/utilities.ts

```diff
@@ -0,0 +1,63 @@
+import { isString } from '@unocss/core'
```

Comment:
I think these basic utility functions can be moved into the `utils` package

Reply:
Yes! We can